terraform/envs/dev/terraform.tfvars.example

```diff
@@ -3,3 +3,5 @@ account_id = ""
 aws_region = ""
 converter_cluster_ami_id = ""
 vpc_cidr_block = ""
+redshift_username = ""
```

Comment:
Where are the tfvars stored now? Is this change made in this PR?

Reply:
In the S3 terraform bucket, together with the tfstate files

Changes:
https://github.com/HumanCellAtlas/matrix-service/commit/0fc0cb4890f99874f86b55916a8dbc8719ccd874

Reply:
got it. This pattern looks familiar ;)